windows-terminal-quake/Program.cs

```diff
@@ -30,10 +32,16 @@ public static void Main(string[] args)
 							FileName = "wt",
 							UseShellExecute = false,
 							RedirectStandardOutput = true,
-							WindowStyle = ProcessWindowStyle.Maximized
+							WindowStyle = ProcessWindowStyle.Maximized,
+							CreateNoWindow = true,
 						}
 					};
 					process.Start();
+					process = Process.GetProcessesByName("WindowsTerminal").First();
+					process.WaitForInputIdle();
+					process.Refresh();
+					Log.Logger.Information("process was started from quake console (to be tested values)");
+					LogProcessInformation(process);

```

Comment:
When I take this part out, it'll actually still show the behaviour like in the video I made.

Perhaps there is a race condition between the process initialising and the toggler trying to get its `mainWindowHandle`?